services/web/src/screens/Shops/index.js

```diff
@@ -1,121 +1,105 @@
-                          Created
-                          <HelpTip title="Created" text="This is the date and time the item was created." />
-                        </Table.HeaderCell>
-                        <Table.HeaderCell textAlign="center">Actions</Table.HeaderCell>
-                      </Table.Row>
-                    </Table.Header>
-                    <Table.Body>
-                      {items.map((item) => {
-                        return (
-                          <Table.Row key={item.id}>
-                            <Table.Cell>
-                              <Link to={`/shops/${item.id}`}>{item.name}</Link>
-                            </Table.Cell>
-                            <Table.Cell>{item.description}</Table.Cell>
-                            <Table.Cell>{formatDateTime(item.createdAt)}</Table.Cell>
-                            <Table.Cell textAlign="center">
-                              <EditShop
-                                item={item}
-                                trigger={<Button style={{ marginLeft: '20px' }} basic icon="edit" />}
-                                onSave={reload}
-                              />
-                              <Confirm
-                                negative
-                                confirmText="Delete"
-                                header={`Are you sure you want to delete "${item.name}"?`}
-                                content="All data will be permanently deleted"
-                                trigger={<Button basic icon="trash" />}
-                                onConfirm={async () => {
-                                  await request({
-                                    method: 'DELETE',
-                                    path: `/1/shops/${item.id}`,
-                                  });
-                                  reload();
-                                }}
-                              />
-                            </Table.Cell>
-                          </Table.Row>
-                        );
-                      })}
-                    </Table.Body>
-                  </Table>
-                )}
-              </Container>
-            );
-          }}
-        </SearchProvider>
-      </AppWrapper>
+      <React.Fragment>
+        {loading ? (
+          <Loader active>Loading</Loader>
+        ) : (error) ? (
+          <React.Fragment>
+            <Breadcrumbs
+              link={<Link to="/shops">Shops</Link>}
+              active="Not Found"
+            />
+            <NotFound message="Sorry that shop wasn't found." />
+          </React.Fragment>
+        ) : (
+          <Switch>
+            <Route path="/shops" component={List} exact />
+            <Route
+              exact
+              path="/shops/:id"
+              render={(props) => (
+                <Overview {...props} {...this.state}  />
+              )}
+            />
+            {/* --- Generator: routes */}
```

Comment:
As discussed: when `bedrock create` is done we want to make sure the output is either generated or stripped from the generator tags

Reply:
this will be up to the generator in the CLI so I'll make a note and resolve here for now